src/Library/demos/Overlay/main.blp

```diff
@@ -0,0 +1,56 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Overlay";
+  description: "Overlay widgets on top of a main widget";
+
+  Adw.Clamp{
```

Comment:
doesn't `StatusPage` already clamp?

Reply:
I think it doesn't clamp enough, otherwise it looks like this

![image](https://user-images.githubusercontent.com/58233418/227777356-10fcafa1-3db4-46d0-820e-437965f71279.png)

Or do you suggest something else?

Reply:
ha I see 

fine as is :+1: